Formula/p/postgresql@17.rb

```diff
@@ -181,8 +181,8 @@ def caveats
     system bin/"initdb", testpath/"test" unless ENV["HOMEBREW_GITHUB_ACTIONS"]
     [bin/"pg_config", HOMEBREW_PREFIX/"bin/pg_config-#{version.major}"].each do |pg_config|
       assert_equal "#{HOMEBREW_PREFIX}/share/#{name}", shell_output("#{pg_config} --sharedir").chomp
-      assert_equal "#{HOMEBREW_PREFIX}/lib/#{name}", shell_output("#{pg_config} --libdir").chomp
-      assert_equal "#{HOMEBREW_PREFIX}/lib/#{name}", shell_output("#{pg_config} --pkglibdir").chomp
+      assert_equal lib.to_s, shell_output("#{pg_config} --libdir").chomp
+      assert_equal lib.to_s, shell_output("#{pg_config} --pkglibdir").chomp
```

Comment:
might need to understand why this got changed, seems a bit fishy

Reply:
Indeed, it feels like sudden change and we have to check it.

Reply:
gonna adopt something similar as https://github.com/Homebrew/homebrew-core/pull/199616